packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandOptionChannelTypesMixin.ts

```diff
@@ -16,15 +16,7 @@ const allowedChannelTypes = [
 
 export type ApplicationCommandOptionAllowedChannelTypes = typeof allowedChannelTypes[number];
 
-const channelTypesPredicate = z.array(
-	z.union(
-		allowedChannelTypes.map((type) => z.literal(type)) as [
-			ZodLiteral<ChannelType>,
-			ZodLiteral<ChannelType>,
-			...ZodLiteral<ChannelType>[]
-		],
-	),
-);
+const channelTypesPredicate = s.array(s.union(...allowedChannelTypes.map((type) => s.literal(type))));
```

Comment:
I wonder in the future if there's any way to use nativeEnum for this too (probably not without remaking the enum here so this is fine)